modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/editor/ClassicEditor.js

```diff
@@ -87,7 +76,7 @@ const ClassicEditor = ({
 	};
 
 	useEffect(() => {
-		setToolbarSet(getToolbarSet(initialToolbarSet));
+		setToolbarSet(initialToolbarSet);
```

Comment:
We can remove this `useEffect`, as `initialToolbarSet` never changes. Even if we keep the prop, it is sufficient to:
```
const [toolbarSet, setToolbarSet] = useState(initialToolbarSet);
```

Reply:
Will do! 👍